app/models/word.rb

```diff
@@ -0,0 +1,6 @@
+class Word < ApplicationRecord
+  validates :category_id, presence: true
```

Comment:
>   **validates :category_id, presence: true**

Please remove this line of code because we already have relationship between WORD and CATEGORY.
So that means it will already check if CATEGORY is present or not. No need to check again.